pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -133,6 +175,27 @@ def anonymise_dataset(  # pylint: disable = inconsistent-return-statements
         An anonymised version of the input DICOM dataset.
     """
 
+    if replacement_strategy is not None:
+        if "replace_values" in replacement_strategy:
+            replace_values = replacement_strategy["replace_values"]
+        if "keywords_to_leave_unchanged" in replacement_strategy:
+            keywords_to_leave_unchanged = replacement_strategy[
+                "keywords_to_leave_unchanged"
+            ]
+        if "delete_private_tags" in replacement_strategy:
+            delete_private_tags = replacement_strategy["delete_private_tags"]
+        if "delete_unknown_tags" in replacement_strategy:
+            delete_unknown_tags = replacement_strategy["delete_unknown_tags"]
+        if "copy_dataset" in replacement_strategy:
+            copy_dataset = replacement_strategy["copy_dataset"]
```

Comment:
As mentioned above, preference here to have this variable be defined as a function input, just to have approaches between the functions be more consistent and less cognitive load on someone reading the code :)

Reply:
Other than copy_dataset, I think the other named parameters have been eliminated now.
I can go back in and eliminate the if statements and just assign as needed, or push the strategy further down.